official/utils/logging/examples_per_second_hook.py

```diff
@@ -0,0 +1,73 @@
+from __future__ import absolute_import
```

Comment:
Can we change the filename here? Naming after the class seems overly specific if we anticipate adding more. Maybe performance_hooks? Or better yet, call this one hooks.py, and name the other one hooks_helper or something to that effect.

Reply:
Also, I think we need the licensing code at the top of each file.